src/Integrations/Integrations/Memcached/MemcachedIntegration.php

```diff
@@ -169,10 +176,9 @@ function (SpanData $span, $args, $retval) use ($integration, $command) {
                 if ($command === 'getMulti') {
                     $span->metrics[Tag::DB_ROW_COUNT] = isset($retval) ? (is_array($retval) ? count($retval) : 1) : 0;
                 }
-                if (!is_array($args[0])) {
-                    $integration->setServerTags($span, $this);
-                }
+                $integration->setServerTags($span, $this);
```

Comment:
This `if` check was a carryover from old code here https://github.com/DataDog/dd-trace-php/commit/9924c1b3a36c10d3ca90a061718d29443da80806, when it was actually required.